roles/confluent.test/molecule/kerberos-rhel/verify.yml

```diff
@@ -59,6 +91,14 @@
         property: security.protocol
         expected_value: SASL_PLAINTEXT
 
+    - import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/ksqldb/ksql-server.properties
+        property: sasl.kerberos.service.name
+        expected_value: kafka-a
```

Comment:
@domenicbove why is the expected value kafka-a should it not just be kafka?

Reply:
So the default value for `kerberos_kafka_broker_primary` is kafka:
```
kerberos_kafka_broker_primary: "{{ (hostvars[ groups['kafka_broker'][0] | default('kafka') ] | default({})) ['kafka_broker_kerberos_principal'] | default('kafka/host@EXAMPLE>COM') | regex_replace('/.*') }}"
```

We need a test where the primary is not the default to make sure the logic works for other primaries. If you look in this scenario's molecule.yml file, the principal is: `kafka_broker_kerberos_principal: "kafka-a/{{inventory_hostname}}.confluent@{{kerberos.realm | upper}}"`

Reply:
Makes sense!